uber/models.py

```diff
@@ -1392,7 +1406,12 @@ def jobs_for_signups(self):
 
         def get_account_by_email(self, email):
             return self.query(AdminAccount).join(Attendee).filter(func.lower(Attendee.email) == func.lower(email)).one()
-            
+
+        def get_attendee_watchentry(self, attendee):
```

Comment:
FYI, saying ``and_`` here is unnecessary.  The default behavior of ``.filter()`` is to ``AND`` the arguments together.  In other words, the following two statements are identical:

```python
q = session.query(Foo).filter(Foo.bar == 5, Foo.baz == 6)
q = session.query(Foo).filter(and_(Foo.bar == 5, Foo.baz == 6))
```